msticpy/auth/cloud_mappings.py

```diff
@@ -5,25 +5,25 @@
 # --------------------------------------------------------------------------
 """Azure Cloud Mappings."""
 import contextlib
+import requests
```

Comment:
We switched to using httpx in place of requests but it's almost 100% drop in replacement.
Also have a couple of support utils for timeouts,  user agent string. (actually that reminds me that I need to implement proxy support across these calls.